### ISSUE TITLE
Could not install the update

### ISSUE THREAD

**Issue by gowtham171** [SudoPlz/sp-react-native-in-app-updates#112]
After downloading can't install the update TIA

**Comment by malikzype** [comment]
Yes, Same here...the app gets downloaded and the status is 11 which indicates it's Downloaded. Can anyone help steps to install the update? Will a restart of the app work?

**Comment by afif-malghani** [comment]
Hey @gowtham171  are you calling the `inAppUpdates.installUpdate();`. When working with flexible updates, you need to install by calling the aforementioned function. It is however not required for Immediate updates.
